packages/bits/src/lib/expander/expander.component.html

```diff
@@ -25,7 +25,7 @@
         </div>
     </div>
     <div class="nui-expander__body"
-         role="list"
+         role="section"
```

Comment:
Is this a valid role? I couldn't find it on the MDN site. I'm not sure what role would be appropriate here. Maybe `document`?

Reply:
![image](https://user-images.githubusercontent.com/75837119/113397448-bc1f8000-93a5-11eb-8ea0-4b9c424d1ef6.png)

from https://www.w3.org/TR/wai-aria-1.1/

Reply:
Oh, I see. It looks like it's an abstract role, though, indicating that it shouldn't be used directly: https://www.w3.org/TR/wai-aria-1.1/#section

This page has the following:
![image](https://user-images.githubusercontent.com/10159693/113627545-4d525900-9629-11eb-86af-ffb45c2801bd.png)

Reply:
I think for expander-body the role is unnecessary. Will removed.